### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
modal component is added.

-releated css you can find in modal.css

screenshots:
modal1
modal2

### CODE REVIEW

**Review by sakleprajwal — APPROVED** [review]
The code looks fine. Please go ahead.

File: documentation/doc.html [hunk 1]
```diff
@@ -689,73 +694,117 @@ <h4>List Item 3</h4>
...
+              <div class="modal-head">
+                <h3>Basic Modal</h3>
+                <h3 id="modal-close">X</h3>
+              </div>
+              <hr class="break-modal" />
+              <div class="modal-body">
+                <h4>Modal Title</h4>
+                <p>modal content...</p>
+                <p>action saggestion...</p>
+                <p>some content...</p>
```
Comment (tusharkandpal) [inline documentation/doc.html:777]: i think you can fill in some mock data here if you want.

**Review by Raghav888 — APPROVED** [review]
Good structure

**Review by tusharkandpal — APPROVED** [review]
seems consistent with others. good to merge!

File: documentation/doc.html (hunk 1 above)
Comment (kotalwar-krishna) [reply documentation/doc.html:777]: yes i can

**Pull request merged**